rero_ils/modules/holdings/jsonschemas/holdings/holding-v0.0.1.json

```diff
@@ -151,6 +154,114 @@
             }
           }
         },
+        "frequency": {
+          "title": "Frequency",
+          "type": "string",
+          "enum": [
+            "Annual",
+            "Biennial",
+            "Bimonthly",
+            "Biweekly",
+            "Completely irregular",
+            "Daily",
+            "Monthly",
+            "Quarterly",
+            "Semiannual",
+            "Semimonthly",
+            "Semiweekly",
+            "Three times a month",
+            "Three times a week",
+            "Three times a year",
+            "Triennial",
+            "Weekly"
```

Comment:
I think here should be the codes (ie values) and not the natural language forms (ie labels).

Reply:
You find the code list here: http://www.rdaregistry.info/termList/frequency/
We already used such code for the data model, in the form of `rdafr:1001`

Reply:
The RDA codes added and the form labels are corrected